1.add '{load static}'

Aleksa-esme opened the pull request:
2.add dynamic urls
3.add 'base.html'
4.add 'test.html' with products context

Aleksa-esme committed:
1.add '{load static}'
2.add dynamic urls
3.add 'base.html'
4.add 'test.html' with products context

vprinz commented:
Александра, здравствуйте!
Отвечая на ваши вопросы:
1. Скачал ваш проект, вроде бы все статики подключаются через `{% load static %}`, даже css. Пришлите скриншот того, какая у вас ошибка и как она выглядит.
2. Да, можно. Вы просто указываете тут вот так `<link rel="stylesheet" type="text/css" href="{% static '<path_to_ sass >' %}">`. В самом файле sass нельзя использовать load static.
3. `<button class="red_button gallery_button" onclick="window.location.href = {% url 'products' %};">Buy</button>`
4. Вам нужно в ваш контроллер products перенести всю информацию о продуктах и передать ее в контекст. Далее в products.html уже в цикле for вывести все эти продукты, используя всего один блок.
5. Ок
6. Я думаю что вы говорите про включенные шаблоны. Мы их разберем на 5 уроке

Как исправите, дайте знать. Проверю еще раз

Aleksa-esme committed:
fixed onclick

Aleksa-esme committed:
fixed onclick
added dictionaries 'discount_first' and 'discount_second' and list 'products' for products.html , lists 'products' and 'team' for index.html
change 'products' cards in products.html index.html with 'for-loop'

Aleksa-esme committed:
fixed my error in 'for-loop' in index.html
